src/js/components/returns/inbound/AddItemsPage.jsx

```diff
@@ -325,7 +325,7 @@ class AddItemsPage extends Component {
   }
 
   async nextPage(formValues) {
-    const status = this.state.inboundReturn.status === StockTransferStatus.PLACED
+    const status = this.state.inboundReturn.status !== StockTransferStatus.PLACED
```

Comment:
I don't one thing here. The ticket mentions putaway, commit message stock transfer and the change is made in the inbound return. Which one is correct?

Reply:
This was supposed to be on the other branch https://github.com/openboxes/openboxes/pull/4478 sorry about that